### DISCUSSION TITLE
Excel Keyboard Commands

### DISCUSSION THREAD

**Discussion by louie-prog** [nvaccess/nvda#13065] [Q&A] [answered]
Good day to all NVDA users. May I know if is it possible to drag the cells in excel using keyboard to increase its size? The NVDA is saying either cropped or over flowing? What does it mean?
Thank you for your response.

**Comment by Qchristensen** [comment, marked as answer]
You can adjust cell size using the ribbon.  There are several options:

1) You can manually set the column width using ALT+H, then O, then W.  Then type a value.
2) You can autofit the width to the contents, by pressing ALT+H, then O, then i.

When manually setting column width, you can use the same command, alt+h, o, w, to simply open the column width dialog and read the existing value - that way when wanting to set a value, you can check other nearby cells to know how large they are.

Autofit sets the cell width to as wide as needed to fit the current contents.  Usually the best way to do this is to first press CONTROL+SPACEBAR to select the whole column, that way autofit will set to the widest cell in the column.  You can do the same thing for row height as well, again these are on the cell formatting option in the home ribbon - alt+h, then o.

If you are interested, all of this and much more is covered in the Microsoft Excel with NVDA training module available at: https://www.nvaccess.org/product/microsoft-excel-training-for-nvda-ebook/

**Comment by louie-prog** [comment]
Glad to know this. Thanks.
